scripts/utils.js

```diff
@@ -41,4 +47,15 @@ module.exports = {
       .replace(/&apos;/g, "’")
       .replace(/&amp;/g, "&")
   ),
+
+  /**
+   * Converts a brand title (as it is seen in simple-icons.json) into a brand
+   * title in HTML/SVG friendly format.
+   * @param {String} brandTitle The title to convert
+   */
+  titleToHtmlFriendly: brandTitle => (
+    brandTitle
+      .replace(/’/g, "&apos;")
+      .replace(/&/g, "&amp;")
```

Comment:
> The `titleToHtmlFriendly` currently in this Pull Request is incorrect, as can be seen from the "Let’s encrypt" example (`Let’s encrypt` -> `Let&apos;s encrypt` -> `Let&amp;apos;s encrypt`). This should definitely be fixed.

```suggestion
      .replace(/&/g, "&amp;")
      .replace(/’/g, "&apos;")
```

Reply:
As I stated in #6522, `letsencrypt` has a wrong title. The source in `data/simple-icons.json` uses the character `’` (`&#8217;`) but the SVG file uses `'` (`&apos;`). You are converting here the character  `’` (`&#8217;`) to `&apos;` (`'`), which is not true.

Reply:
@mondeja, this suggestion is just about switching these two statements around, it's not my suggestion that replaces `’` by `&apos;`, that is how this function works as of 58a95c42f41d37c85e0fdc25009d328d749d9515. I agree that we shouldn't replace `’` (or `'` for that matter) by `&apos` but I just wanted to have this separate comment that `/&/g` always needs to be replaced first. (This may or may not be rendered obsolete by #6522, but I wanted to make sure we don't forget about it if it isn't rendered obsolete.)

```suggestion
      .replace(/&/g, "&amp;")
      .replace(/’/g, "&#8217;")
```

And as I already [said here](https://github.com/simple-icons/simple-icons/pull/6522#issuecomment-920704634) - at least at the time of adding `letsencrypt` - it's title should use `’`/`&#8217;`, so the SVG `<title>` has to be changed.

Reply:
OK, but anyways it seems to me that this implementation is wrong because the `&` characters of an already encoded input would be also replaced, regardless of the order of replacements. I think that this should be addressed in #6522.

Reply:
But this function is intended to convert not-encoded titles (i.e the ones in the data file) to html-encoded (`<title>`) values

Reply:
Yes, but it could be safer. I want to replace this function by a safer converter like:

```js
const newHtmlFriendlyToTitle = (htmlFriendlyTitle) => {
  return htmlFriendlyTitle.replace(
    /&(amp|quot|#039|apos|lt|gt);/g,
    (_, rep) => {
      return {amp: '&', quot: '"', '#039': "'", apos: "'", lt: '<', gt: '>'}[rep]
    }
  );
};

console.log(newHtmlFriendlyToTitle("Hello &amp; world"));  // Hello & world
console.log(newHtmlFriendlyToTitle("Hello & world"));      // Hello & world
console.log(newHtmlFriendlyToTitle("Hello world &"));      // Hello world &

// Actual implementation
const htmlFriendlyToTitle = (htmlFriendlyTitle) => {
  return htmlFriendlyTitle
    .replace(/&/g, "&amp;")
    .replace(/’/g, "&#8217;");
}

console.log(htmlFriendlyToTitle("Hello &amp; world"));  // Hello &amp;amp; world
console.log(htmlFriendlyToTitle("Hello & world"));      // Hello &amp; world
console.log(htmlFriendlyToTitle("Hello world &"));      // Hello world &amp;
```

You know, it would be necessary also to handle the cases where there were decimal or hexadecimal representations. But anyways, I suggest to discuss this in #6505.

Reply:
Are we going to include every single unicode character in this function?

```
.replace(/’/g, "&#8217;")
``` 

This implementation makes no sense, Please, could we discuss this in #6522?

Reply:
To reiterate, I want to avoid a situation where we decide not to merge #6522 and forget about this error in this Pull Request. I agree the suggestion should change in the context of #6522, the current suggestion is in the context of the current state of this Pull Request.

Reply:
> I want to avoid a situation where we decide not to merge #6522 and forget about this error in this Pull Request.

In that case, I'm going to leave this conversation unresolved just to remember about it :+1: